Documentation/ApiOverview/SiteHandling/SiteSettingsEditor.rst

```diff
@@ -0,0 +1,91 @@
+..  include:: /Includes.rst.txt
+..  index:: Site handling; Site settings editor
+..  _site-settings-editor:
+
+====================
+Site settings editor
+====================
+
+..  versionadded:: 13.3
+    The site setting editor has been introduced as backend module
+    :guilabel:`Sites > Settings`.
```

Comment:
To be very correct it's "Site Management -> Settings"

Reply:
good catch!